# List of potential ditch sites as determined by the ditch site selection (DSS) manager

nasa_s2d/DitchSite[] ditch_sites